src/main/java/duke/commands/DeleteCommand.java

```diff
@@ -0,0 +1,34 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.Task;
+import duke.TaskList;
+import duke.Ui;
+
+public class DeleteCommand extends Command {
+
+    private int deleteNumber;
```

Comment:
Maybe you can consider renaming this variable to `indexToBeDeleted` to make its purpose clearer.